Tests/Controller/Annotations/AbstractScalarParamTest.php

```diff
@@ -98,4 +98,11 @@ public function testConstraintsTransformWhenParamIsAnArray()
             new Constraints\NotNull(),
         ))), $this->param->getConstraints());
     }
+
+    public function testArrayWithNoConstraintsDoesNotCreateInvalidConstraint()
```

Comment:
@Ener-Getick I just pushed up a change to test that it's creating the constraint with an empty `constraints` key, would you prefer I just remove the test entirely?